### ISSUE TITLE
XML Sitemap - indexes should be reverse chronological order

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#2217]
Reported here: https://wordpress.org/support/topic/bugs-with-the-sitemap-please-fix-them/

> Newest posts are always written in the first sitemap. For example, when sitemap is split in 5 chunks, 1000 posts per split, in sitemap5.xml 1000 in 4 1000, 3 1000, 2 1000, Sitemap1 will have the newest posts but when that fills up with 1000, sitemap1 will change name into sitemap 2, 2 will change name into 3, 4 into 5 and so on. This is a very bad practice which according to Google themselves causes deindexing:
https://www.seroundtable.com/google-dynamic-sitemap-file-names-25187.html
Please fix this. YOAST SEO has figured it out, its the only sitemap out there which has the oldest 1000 posts in sitemap1 and gradually the newest posts in the last sitemap, lets say 5. After 5 is full with 1000 newest posts, that will remain the same, unchanged, FOREVER, and sitemap6 will be created and filled with the newest posts. This is the most intelligent way a sitemap should behave.
In short, please put the newest posts at the bottom and the oldest on top or simply just give an option in the settings and let users choose the order.

**Comment by vikingrunes** [comment]
I can't believe this isn't getting any traction and discussion. This is actually true and very important. Sub-sitemaps can't change names all the time, Google will go crazy. Its actually important to start with the oldest posts first and gradually move to the newest. Isn't milestone 3.0 kind of too far ahead? Can't this be implemented in the next update? Shouldn't be that complicated to do.

**Comment by michaeltorbert** [comment]
3.0 is the next release. It’s been marked as high priority.

**Comment by vikingrunes** [comment]
ohh sorry, I didn't know 3.0 is the next update, I thought there would be 2.13, 2.14 and so on, LOL. Well that's great news, I'm really happy this is getting taken care of. Cheers

**Comment by contactashish13** [comment]
PR https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/2263
